packages/flutter/lib/src/material/switch.dart

```diff
@@ -595,24 +609,24 @@ class Switch extends StatelessWidget {
 
     switch (_switchType) {
       case _SwitchType.material:
-        effectiveActiveThumbColor = activeColor;
+        effectiveActiveThumbColor = activeThumbColor;
```

Comment:
I think here should be:
```suggestion
        effectiveActiveThumbColor = activeThumbColor ?? activeColor;
```

Reply:
Got it. Thanks for the update!